src/csharp/examples/server/API/Persistence/UserModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace SessionlessExample.Server.Models
+
+{
+    public class UserModel(string uuid, string pubKey)
```

Comment:
`record` works best for this kind of models where data is the focus and mutability is not preferred (record will improve `Equals`, `GetHashCode`, and `==` operators for that matter)
```csharp
public record UserModel(string uuid, string pubKey);
```